docs/archive-data.md

```diff
@@ -136,58 +100,60 @@ namespaceDefaults:
+| `namespaceDefaults.archival.history.URI`       | Valid URI              | The value must be a URI of the file store location and match a schema that correlates to a provider. |
 
-At the cluster level, `archival.history.state` must be set to `"enabled"` for Archival to work at the Namespace level for Even Histories. Once enabled at the cluster level, a Namespace can safely switch between `enabled` and `disabled` states.
-
-Archival is supported in global Namespaces (Namespaces spanning multiple clusters). When Archival is running in a global Namepsace, it will first run on the active cluster and then some time later it will run on the standby cluster. Before archiving, a history check is done to see what has been previously uploaded.
+### Namespace creation
 
-## Archive reads
- 
-To retrieve archived Event Histories you will need the Workflow Id and the Run Id:
+While Archival is configured at the cluster level, it operates independently within each Namespace. When a Namespace is created, if an Archival URI is not specified then the Namespace will use the value of `defaultNamespace.archival.history.URI` from the `config/development.yaml` file. The Archival URI can not be changed once the Namespace is created. Each Namespace supports a single Archival URI, but each Namespace can use a different URI (archiver). A Namespace can safely switch Archival between `enabled` and `disabled` states as long as Archival is enabled at the cluster level.
 
-```bash
-$ ./temporal --ns <namespace> wf show --wid <workflowId> --rid <runId>
-```
+Archival is supported in global Namespaces (Namespaces spanning multiple clusters). When Archival is running in a global Namepsace, it will first run on the active cluster and then some time later it will run on the standby cluster. Before archiving, a history check is done to see what has been previously archived.
 
-:::tip
+### Test setup
 
-Want to test this feature locally? Start by running a Temporal server:
+To test Archival locally, start by running a Temporal server:
 
 ```bash
-$ ./temporal-server start
+./temporal-server start
 ```
 
 Then register a new Namespace with Archival enabled. 
 
 ```bash
-$ ./tctl --ns samples-namespace namespace register --gd false --history_archival_status enabled --retention 0
+./tctl --ns samples-namespace namespace register --gd false --history_archival_status enabled --retention 0
 ```
 
 Next, run a sample workflow such as the [helloworld temporal sample](https://github.com/temporalio/temporal-go-samples/tree/master/helloworld).
 
-Once the sample execution has finished, you can view the archived Event Histories by copying the `workflowId` and `runId` of the completed Workflow from the log output and running:
+Once execution has finished Archival should occur.
+
+### Read archives
```

Comment:
It's weird title. It's the correct style but 'read archives' actually means something so its confusing